pkg/asset/installconfig/ovirt/credentials.go

```diff
@@ -97,12 +97,18 @@ func (c *clientHTTP) checkURLResponse() error {
 // askPassword will ask the password to connect to Engine API.
 // The password provided will be added in the Config struct.
 // If an error happens, it will ask again username for users.
-func askPassword(c *Config) error {
+func askPassword(c *Config, hintMsg string) error {
```

Comment:
I believe we don't need hintMsg as param for a askPassword() function.

Reply:
No pb I'll remove it